samples/testing-frameworks/appium/client-side/csharp/biometric-sample-test/BitbarBiometricSample/TestBiometricsIOS.cs

```diff
@@ -10,9 +10,9 @@ public class TestBiometricsIOS : BaseBiometricTest
         public void InitDriver()
         {
             //Platform dependent
-            capabilities.AddAdditionalCapability("platformName", "iOS");
-            capabilities.AddAdditionalCapability("deviceName", "iPhone device");
-            capabilities.AddAdditionalCapability("automationName", "XCUITest");
+            capabilities.AddAdditionalCapability("appium:platformName", "iOS");
```

Comment:
according to https://appium.io/docs/en/2.0/guides/caps/
platformName should be without prefix